spec/components/schema/tour.yaml

```diff
@@ -119,6 +138,11 @@ components:
           type: array
           items:
             $ref: "../commons/objects.yaml#/components/schemas/Durations"
+        availabilities:
```

Comment:
Have we checked with Oliver if this is okay to add ?

Reply:
As discussed in standup, let's keep as it is today